### ISSUE TITLE
Add option to control friction when slide=true

### ISSUE THREAD

**Issue by R2D221** [skidding/dragdealer#91]
I'm using DradDealer for a gallery similar to the cars demo. However, I've noted that swiping is too slow, because the range for sliding is low.

I've found in the source code the following commands:
      target[0] += ratioChange[0] * 4;
      target[1] += ratioChange[1] * 4;

The number 4 is the value I'm looking for, and changing its value to something bigger gives me the effect I want.

It would be nice that instead of editing the source code, it was provided as a customizable option.

**Comment by skidding** [comment]
Agreed. Feel free to contribute if you have an idea of how to call it. Can't think of a good, simple name right now to be honest :thought_balloon:

**Comment by kaigth** [comment]
I have added inertia to my most recent PR for this project, this allows you to control how fast the throw action occurs and can now be sped up or reduced.
